config/data/offence_synonyms.yml

```diff
@@ -3,3 +3,7 @@ offence_synonyms:
   synonyms: [gbh]
 - regex: !ruby/regexp /section 18/i
   synonyms: [s18]
+- regex: !ruby/regexp /anti-social behaviour/i
+  synonyms: [asb]
+- regex: !ruby/regexp /criminal behaviour order/i
+  synonyms: [cbo]
```

Comment:
We have Section 4 and Section 4A that I imagine could well be S4 and S4A - worth adding, or researching that with providers?

Actual bodily harm commonly is ABH too - One of the offence names includes it, but one other unhelpfully doesn't

Reply:
Oh nice, `ABH` I can add that one too in this PR.

Yep I think the best we could do is work with providers on what they usually refer as shortcuts or abbreviations, is quite difficult for us to know in advance.

Reply:
Yeah, very difficult. I also imagine they don't know they do it! S18 is so normal to them they might not consider it an abbreviation